qlib/rl/order_execution/simulator_qlib.py

```diff
@@ -2,3 +2,199 @@
 # Licensed under the MIT License.
 
 """Placeholder for qlib-based simulator."""
+import copy
+from typing import Callable, Generator, List, Optional, Tuple, Union
+
+import pandas as pd
+from gym.vector.utils import spaces
+
+from qlib.backtest import get_exchange
+from qlib.backtest.account import Account
+from qlib.backtest.decision import Order, TradeRange, TradeRangeByTime
+from qlib.backtest.executor import BaseExecutor
+from qlib.backtest.utils import CommonInfrastructure
+from qlib.config import QlibConfig
+from qlib.rl.interpreter import ActionInterpreter
+from qlib.rl.order_execution.from_neutrader.config import ExchangeConfig
+from qlib.rl.order_execution.from_neutrader.executor import RLNestedExecutor
+from qlib.rl.order_execution.from_neutrader.feature import init_qlib
+from qlib.rl.order_execution.from_neutrader.state import SAOEEpisodicState
```

Comment:
I think you can reuse the SAOEState from `simulator_simple`. `SAOEEpisodicState` is already deprecated.